e2e-tests/playwright/e2e/plugins/ocm.spec.ts

```diff
@@ -32,6 +32,18 @@ test.describe.serial("Test OCM plugin", () => {
   });
   test("Navigate to Clusters and Verify OCM Clusters", async () => {
     await uiHelper.openSidebar("Clusters");
+    const expectedPath = "/ocm";
+
+    // Wait for the expected path in the URL
+    await page.waitForURL(`**${expectedPath}`, {
+      waitUntil: "domcontentloaded", // Wait until the DOM is loaded
+      timeout: 10000, // Set timeout to 10 seconds
+    });
+
+    expect(page.url()).toContain(expectedPath);
+
+    await uiHelper.verifyHeading("Your Managed Clusters");
+    await uiHelper.verifyText(clusterDetails.clusterName);
```

Comment:
This already being verified in bellow line `await uiHelper.verifyRowInTableByUniqueText(clusterDetails.clusterName`